### PR TITLE
[pravila00-enum-string] New port

### PR DESCRIPTION
**Pull request by Pravila00**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [ ] <del> The name of the port matches an existing name for this component on https://repology.org/ if possible, and/or is strongly associated with that component on search engines. </del> 
- [ ] <del> Optional dependencies are resolved in exactly one way. For example, if the component is built with CMake, all `find_package` calls are REQUIRED, are satisfied by `vcpkg.json`'s declared dependencies, or disabled with [CMAKE_DISABLE_FIND_PACKAGE_Xxx](https://cmake.org/cmake/help/latest/variable/CMAKE_DISABLE_FIND_PACKAGE_PackageName.html) </del> 
- [x] The versioning scheme in `vcpkg.json` matches what upstream says.
- [x] The license declaration in `vcpkg.json` matches what upstream says.
- [x] The installed as the "copyright" file matches what upstream says.
- [x] The source code of the component installed comes from an authoritative source.
- [ ] <del> The generated "usage text" is accurate. See [adding-usage](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/examples/adding-usage.md) for context. </del>
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is in the new port's versions file.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/pravila00-enum-string/vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "pravila00-enum-string",
+  "version-date": "2023-16-10",
```
Comment (FrankXie05) [inline ports/pravila00-enum-string/vcpkg.json:3]:
```suggestion
  "version-date": "2023-10-16",
```
Note the format：
https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/users/versioning.md#version-date

File: ports/pravila00-enum-string/portfile.cmake [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO Pravila00/enum-string
+    REF 3eec46f5520c287ee46692ea1d41833cbe0d80f2
+    SHA512 5b29c27b8ceb358bae5a2b4fb5d198b7b4cd8a7c9926bd7685c27650da5cda9f6dc85b6a9694fe151d03b22e3230d5f3faf9143e865dffc0795b2952fca5fc0f
+    HEAD_REF main
+)
+
+file(COPY "${SOURCE_PATH}/EnumString.h" DESTINATION "${CURRENT_PACKAGES_DIR}/include/enum-string")
+
+vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE")
```
Comment (FrankXie05) [inline ports/pravila00-enum-string/portfile.cmake:11]:
```suggestion
vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE")

```
Empty line in the end.

File: ports/pravila00-enum-string/portfile.cmake [hunk 3]
```diff
@@ -0,0 +1,11 @@
+vcpkg_from_github(
```
Comment (FrankXie05) [inline ports/pravila00-enum-string/portfile.cmake:1]:
```suggestion
# Header-only library

vcpkg_from_github(
```

File: ports/pravila00-enum-string/portfile.cmake [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO Pravila00/enum-string
+    REF 3eec46f5520c287ee46692ea1d41833cbe0d80f2
+    SHA512 5b29c27b8ceb358bae5a2b4fb5d198b7b4cd8a7c9926bd7685c27650da5cda9f6dc85b6a9694fe151d03b22e3230d5f3faf9143e865dffc0795b2952fca5fc0f
+    HEAD_REF main
+)
+
+file(COPY "${SOURCE_PATH}/EnumString.h" DESTINATION "${CURRENT_PACKAGES_DIR}/include/enum-string")
```
Comment (FrankXie05) [inline ports/pravila00-enum-string/portfile.cmake:9]:
```suggestion
file(COPY "${SOURCE_PATH}/EnumString.h" DESTINATION "${CURRENT_PACKAGES_DIR}/include/${PORT}")
```
I don't think it's necessary to have a port that starts with the author's name.

**Review by FrankXie05 — CHANGES_REQUESTED** [review]

**Comment by FrankXie05** [comment]
Note: I will be converting your PR to draft status. When you respond, please revert to "ready for review". 
          That way, I can be aware that you've responded since you can't modify the tags.

File: ports/pravila00-enum-string/portfile.cmake (hunk 4 above)
Comment (Pravila00) [reply ports/pravila00-enum-string/portfile.cmake]: Sure, it is fine with me. I've just removed the author's name from the beginning :)

**Comment by Pravila00** [comment]
This should be for review again @FrankXie05, thanks a lot!

**Review by FrankXie05 — DISMISSED** [review]

**Comment by JavierMatosD** [comment]
The port LGTM, but since this is a new library and it doesn't show up on repology, could you change the port's name? We typically request that newer libraries add a prefix to the name.

**Comment by Pravila00** [comment]
@JavierMatosD I have added again the prefix for the port, let me know if it looks good to you. Thanks!

**Pull request merged**